src/app/game/simon/page.tsx

```
@@ -0,0 +1,441 @@
+              Strict Mode (mistake = game over)
+            </label>
+            <button className={styles.startButton} onClick={startGame}>
+              Start Game
+            </button>
+          </div>
+        </div>
+      )}
+
+      {gameState !== 'idle' && (
+        <>
+          <div className={styles.status} role="status" aria-live="polite">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)